src/friendly-iframe-embed.js

```diff
@@ -452,6 +454,144 @@ export class FriendlyIframeEmbed {
       this.visibilityObservable_.fire(this.visible_);
     }
   }
+
+  /**
+   * @return {!HTMLBodyElement}
+   * @visibleForTesting
+   */
+  getBodyElement() {
+    return /** @type {!HTMLBodyElement} */ (
+        (this.iframe.contentDocument || this.iframe.contentWindow.document)
+            .body);
+  }
+
+  /**
+   * @return {!./service/vsync-impl.Vsync}
+   * @visibleForTesting
+   */
+  getVsync() {
+    return vsyncFor(this.win);
```

Comment:
I think you're stubbing at the right level. The problem is there's no good (ES6-compliant) way to stub imported functions. Using a wrapper function (like you are here) or getting the service in the constructor and stubbing the ivar are the only ways to do it AFAIK.